src/components/assets/Erc20Currency copy.vue

```diff
@@ -0,0 +1,125 @@
+<template>
```

Comment:
Please fix the name of this file

Reply:
Fixed in this commit.
https://github.com/AstarNetwork/astar-apps/pull/397/commits/e626c3c0e753c7ec061b2292feaaab8be186c1fd